VideoLocker/res/layout/row_discussion_comments_response.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.v7.widget.CardView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_marginLeft="@dimen/discussion_responses_standard_margin"
+    android:layout_marginRight="@dimen/discussion_responses_standard_margin"
+    android:layout_marginTop="@dimen/discussion_responses_standard_margin"
+    android:layout_marginBottom="@dimen/discussion_responses_standard_margin"
+    app:cardCornerRadius="@dimen/edx_box_radius"
+    app:cardPreventCornerOverlap="false"
+    app:cardUseCompatPadding="true">
+
+    <LinearLayout
+        android:id="@+id/row_discussion_comment_layout"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:orientation="vertical"
+        android:padding="@dimen/discussion_responses_box_padding">
+
+        <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:orientation="horizontal">
+
+            <include layout="@layout/discussion_user_profile_row" />
+
+            <Space
+                android:layout_width="0dp"
+                android:layout_height="0dp"
+                android:layout_weight="1" />
+
+            <TextView
+                android:id="@+id/discussion_comment_count_report_text_view"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:background="?android:selectableItemBackground"
+                android:clickable="true"
+                android:drawablePadding="5dp"
+                android:padding="@dimen/discussion_responses_box_padding"
+                android:textColor="@color/edx_grayscale_neutral_base"
+                android:textSize="@dimen/edx_xxx_small"
+                tools:text="8 comments" />
+
+        </LinearLayout>
+
+        <TextView
+            android:id="@+id/discussion_comment_body"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:textColor="@color/edx_grayscale_neutral_dark"
+            android:textSize="@dimen/edx_x_small"
+            tools:text="This is the text of a response." />
+    </LinearLayout>
```

Comment:
Again, note that the inner layout here (excluding the root `CardView`), seems to be functionally identical to the `discussion_responses_thread_row` layout. There are small differences in the text view font sizes etc., but I think they're supposed to be identical in the new design. Therefore, this layout may also be conceivably centralized in a shared layout file for the most part, that is included in both places.